### ISSUE TITLE
Expose `storage-encrypted` parameter for `rds2.create_db_instance`

### ISSUE THREAD

**Issue by ivanistheone** [boto/boto#3323]
The current boto2 api doesn't seem up to date with the RDS possibilities, in particular regarding on-disk encryption.

What is the current recommended way for scripting RDS provisioning? Does boto3 have better coverage of the full RDS API?

**Comment by mtdowling** [comment]
Yes, Boto 3 supports this the entire low level RDS API (and virtually every other API offered by AWS): https://boto3.readthedocs.org/en/latest/reference/services/rds.html

**Comment by ivanistheone** [comment]
Hi Michael, thx for your response. I've since switched to using the RDS functions in `botocore` (a little camel-casey, but works great). Is there any difference between `boto3` and `botocore`? From what I can tell, it seems to be the same thing (i.e. both are the same underlying json APIs).

Is there anything `boto3` provides that is not in `botocore`? (I'm using RDS, ELB, and route53).

**Comment by jamesls** [comment]
The main thing boto3 provides over botocore is higher level abstractions ("resources") as well as some high level abstractions.  If you're just using the client's instead of resources (`session.create_client()` and `boto3.client()`), then there's not much difference.  Given that rds does not have a resource model in boto3, there's not much difference in this specific case.
